include/class_loader/class_loader.hpp

```diff
@@ -188,40 +186,62 @@ class ClassLoader
 
   /**
    * @brief Gets the full-qualified path and name of the library associated with this class loader
+   * @return the full-qualified path and name of the library
    */
   CLASS_LOADER_PUBLIC
   const std::string & getLibraryPath() const;
 
   /**
-   * @brief  Indicates if a library is loaded within the scope of this ClassLoader. Note that the library may already be loaded internally through another ClassLoader, but until loadLibrary() method is called, the ClassLoader cannot create objects from said library. If we want to see if the library has been opened by somebody else, @see isLibraryLoadedByAnyClassloader()
+   * @brief  Indicates if a library is loaded within the scope of this ClassLoader. Note that the
```

Comment:
```suggestion
   * @brief Indicates if a library is loaded within the scope of this ClassLoader.
   *
   * Note that the
```

Reply:
(check the other lines for extra spaces)

Reply:
Added some extra lines: https://github.com/ros/class_loader/pull/156/commits/3239f3e30a621a4e3c3c3d909c11edfdb2f79084